package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@reactioncommerce/api-plugin-accounts",
   "description": "Accounts plugin for the Reaction API",
-  "version": "0.0.0-development",
+  "version": "1.0.0",
```

Comment:
Just in case this one slips the radar - versions should stay at `0.0.0-development`